cves/CVE-2013-2844.yml

```
@@ -165,8 +183,11 @@ lessons:
     applies: 
     note: 
   distrust_input:
-    applies: 
-    note: 
+    applies: true
+    note: Due to the ability of external networks to free the CSS renderer,
```

Please indent these lines so the CI passes.
```suggestion
    note: |
      Due to the ability of external networks to free the CSS renderer,
      the memory pointers may be freed and reused again outside of the control
      of the Chrome application. This vulnerability can lead to potential crashes
      or denial of service vulnerabilities.
```